templates/errors.rb

```diff
@@ -23,7 +23,7 @@
   Net::SMTPAuthenticationError,
   Net::SMTPServerBusy,
   Net::SMTPUnknownError,
-  TimeoutError
+  Timeout::Error
```

Comment:
Should we strive to adhere to our style-guide as we touch lines? Eventually, this file would be inline with our preferred style, as opposed to using stale styles.